### PR TITLE
added api documentation, event recipients in sendEventReminder route

### PR DESCRIPTION
**Pull request by sohan2410**
event
event2

### CODE REVIEW

File: server/common/api.yml [hunk 1]
```diff
@@ -651,6 +651,41 @@ paths:
...
+    post:
+      tags:
+        - Admin
+      servers:
+        - url: /
+      description: Send event Reminder
+      security:
+        - Bearer: []
+      requestBody:
+        description: 2D array of Recipients, Text, Event Name and Link for the event
```
Comment (GunaShekar02) [inline server/common/api.yml:664]: I think recipients is just an array, not a 2D one.

File: server/common/api.yml [hunk 2]
```diff
@@ -1005,6 +1040,36 @@ components:
...
           example: [[2018IPG-000, 2018IPG-001, 2018IPG-002], [2018IPG-003]]
 
+    SendEventReminderBody:
+      type: object
+      required:
+        - eventRecipients
+        - text
+        - eventName
+        - link
+        - winners
```
Comment (GunaShekar02) [inline server/common/api.yml:1050]: I think winners shouldn't be here.

File: server/common/api.yml [hunk 3]
```diff
@@ -1005,6 +1040,36 @@ components:
...
+      properties:
+        eventRecipients:
+          type: array
+          items:
+            description: 2D Array of recipients
```
Comment (GunaShekar02) [inline server/common/api.yml:1055]: This should be just an array too, not 2D.

File: server/common/api.yml [hunk 4]
```diff
@@ -1005,6 +1040,36 @@ components:
...
+            type: string
+            items:
```
Comment (GunaShekar02) [inline server/common/api.yml:1057]: The inner items is used when it's a 2D array, we should rather have something like,
```
eventRecipients:
  type: array
  items:
    type: string
    description: Email Recipients
    example: bcs_201831@iiitm.ac.in
  example: ['email1', 'email2', 'email3]
```

**Review by GunaShekar02 — APPROVED** [review]
Looks good overall! Just left a few minor comments.

File: server/common/api.yml (hunk 4 above)
Comment (GunaShekar02) [reply server/common/api.yml:1056]: I think this hasn't been fixed yet, could you please look into it?

**Review by GunaShekar02 — CHANGES_REQUESTED** [review]

File: server/common/api.yml (hunk 4 above)
Comment (sohan2410) [reply server/common/api.yml:1056]: Could you please re-check because I've made changes in previous commit

Comment (GunaShekar02) [reply server/common/api.yml:1056]: I did, currently it looks like this:-
```
        eventRecipients:
          type: array
          items:
            description: Array of recipients
            type: string
            items:
              type: string
              description: Email Recipients
              example: bcs_201831@iiitm.ac.in
          example:
            [' bcs_201831@iiitm.ac.in', 'gunashekherproddatoori@gmail.com', 'aasf.iiitmg@gmail.com']
```
whereas it should look like this:-
```
        eventRecipients:
          type: array
          description: Array of recipients
          items:
            type: string
            description: Email Recipients
            example: bcs_201831@iiitm.ac.in
```

**Review by GunaShekar02 — CHANGES_REQUESTED** [review]

**Comment by sohan2410** [comment]
done

**Pull request merged**